### ISSUE TITLE
page 1402 "Purchase No. Series Setup" - SetFieldsVisibility procedure event request

### ISSUE THREAD

**Issue by SYL-denisg** [microsoft/ALAppExtensions#26630]
### Describe the request

Hello, we would like a trailing event added to the SetFieldsVisibility procedure in Page 1420 "Purchase No. Series Setup", that allows us to run some additional code.

Thanks,

Denis

```
page 1402 "Purchase No. Series Setup"
.....
    procedure SetFieldsVisibility(DocType: Option Quote,"Order",Invoice,"Credit Memo","Blanket Order","Return Order")
    begin
        QuoteNosVisible := (DocType = DocType::Quote);
        BlanketOrderNosVisible := (DocType = DocType::"Blanket Order");
        OrderNosVisible := (DocType = DocType::Order);
        ReturnOrderNosVisible := (DocType = DocType::"Return Order");
        InvoiceNosVisible := (DocType = DocType::Invoice);
        CreditMemoNosVisible := (DocType = DocType::"Credit Memo");

        OnAfterSetFieldsVisibility(DocType);   // NEW EVENT
    end;

    // NEW EVENT definition, please note the Sender option is set
    [IntegrationEvent(true, false)]
    local procedure OnAfterSetFieldsVisibility(DocType: Option Quote,"Order",Invoice,"Credit Memo","Blanket Order","Return Order")
    begin
    end;
}

```

### Additional context

We have implemented an additional purchase document type (see #26629) and can use this event to help control some visibility issues in parallel to how you do it.
Internal work item: [AB#538373](https://dynamicssmb2.visualstudio.com/1fcb79e7-ab07-432a-a3c6-6cf5a88ba4a5/_workitems/edit/538373)

**Comment by JesperSchulz** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. Please do not reply to this, as we do not monitor closed issues. If you have follow-up questions or requests, please create a new issue where you reference this one.

Build ID: 20959.
